src/geometry/transform.rs

```diff
@@ -0,0 +1,191 @@
+use crate::image::{Image, ImageSize};
+use crate::resize::{interpolate, meshgrid, InterpolationMode};
+use anyhow::Result;
+use ndarray::stack;
+
+type AffineMatrix = (f32, f32, f32, f32, f32, f32);
```

Comment:
For small matrix multiplication I was planning to use the nalgebra crate which seems well optimizadles for that